server.py

```diff
@@ -1,32 +1,36 @@
 from flask import Flask, Response, request
 from hashlib import sha1
+from raven.contrib.flask import Sentry
 import base64
 import hmac
 import os
 import requests
 
-app = Flask(__name__)
-app.debug = os.getenv('DEBUG', '') == 'True'
+def create_app():
+    app = Flask(__name__)
+    app.debug = os.getenv('DEBUG', '') == 'True'
+    if os.getenv('SENTRY_DSN', ''):
```

Comment:
hmm, i guess we don't need the ```, '')``` part?
https://docs.python.org/3/library/os.html#os.getenv
so i suggest
```python
    app.debug = os.getenv('DEBUG') == 'True'
    if os.getenv('SENTRY_DSN'):
```